Entity/Attribute.php

```diff
@@ -57,6 +57,10 @@ public function getId()
      */
     public function setValue($value)
     {
+        if (is_array($value)) {
+            $value = serialize($value);
```

Comment:
It's a bunch of scalar values, so implode / explode might be enough for the job, what do you think ?